packages/react/src/index.tsx

```diff
@@ -125,7 +127,18 @@ export function useTranslations<
       return '';
     }
 
-    return translationsObject[key].format(params);
+    const result = translationsObject[key].format(params);
+
+    if (Array.isArray(result)) {
+      for (let i = 0; i < result.length; i++) {
+        const item = result[i];
+        if (typeof item === 'object' && !item.key && isValidElement(item)) {
+          result[i] = cloneElement(result[i], { key: i });
```

Comment:
Should we namespace the key? To ensure it doesn't clash with others?
E.g. __vocab-1

Reply:
Sure